student_school_DO/hw_2/forTask3

```diff
@@ -0,0 +1,1357 @@
+﻿
```

Comment:
Replace all data files to different folder, for example "student_school_DO/hw_2/**data**/forTask3"